src_web/kamaho-shokusu/tests/TestCase/Policy/TReservationInfoVulnerabilityTest.php

```
@@ -0,0 +1,225 @@
+}
+
+class TestIdentity implements IdentityInterface
+{
+    /**
+     * @param array<string, mixed> $data
+     */
+    public function __construct(private array $data)
+    {
+    }
+
+    public function can(string $action, mixed $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$action'. (undefined)

(UnusedFormalParameter)